### PR TITLE
Fix desired trajectory waypoint

### PR DESCRIPTION
**Pull request by mrivi**
The first waypoint should contain the desired velocity setpoint not the current vehicle velocity.

### CODE REVIEW

**Comment by dagar** [comment]
Check CI failure (trivial style issue) and rebase on master.

File: src/modules/mc_pos_control/mc_pos_control_main.cpp [hunk 1]
```diff
@@ -1007,16 +1007,18 @@ MulticopterPositionControl::failsafe(vehicle_local_position_setpoint_s &setpoint
...
 	_traj_wp_avoidance_desired.timestamp = hrt_absolute_time();
 	_traj_wp_avoidance_desired.type = vehicle_trajectory_waypoint_s::MAV_TRAJECTORY_REPRESENTATION_WAYPOINTS;
 
 	states.position.copyTo(_traj_wp_avoidance_desired.waypoints[vehicle_trajectory_waypoint_s::POINT_0].position);
-	states.position.copyTo(_traj_wp_avoidance_desired.waypoints[vehicle_trajectory_waypoint_s::POINT_0].velocity);
-	states.position.copyTo(_traj_wp_avoidance_desired.waypoints[vehicle_trajectory_waypoint_s::POINT_0].acceleration);
+	_traj_wp_avoidance_desired.waypoints[vehicle_trajectory_waypoint_s::POINT_0].velocity[0] = setpoint.x;
+	_traj_wp_avoidance_desired.waypoints[vehicle_trajectory_waypoint_s::POINT_0].velocity[1] = setpoint.y;
+	_traj_wp_avoidance_desired.waypoints[vehicle_trajectory_waypoint_s::POINT_0].velocity[2] = setpoint.z;
+	states.acceleration.copyTo(_traj_wp_avoidance_desired.waypoints[vehicle_trajectory_waypoint_s::POINT_0].acceleration);
```
Comment (Stifael) [inline src/modules/mc_pos_control/mc_pos_control_main.cpp]: I think it should be `setpoint.vx`, `setpoint.vy` and `setpoint.vz`.

**Comment by mrivi** [comment]
@dagar fixed

**Comment by dagar** [comment]
Rebased

**Pull request merged**